macbuild.tool

```diff
@@ -78,6 +79,13 @@ DEPURLS=(
   'https://github.com/acidanthera/DuetPkg'
 )
 DEPBRANCHES=('master' 'master' 'master')
+
+export ARCHS
+export SELFPKG
+export DEPNAMES
+export DEPURLS
+export DEPBRANCHES
```

Comment:
Why is that needed?

Reply:
Basically same as local variables item.

The declared variables are not actually used anywhere in this script file but are for efibuild.sh which is run in a subshell.

While things work as it, technically should be exported in such instances. Again, something that is not an issue here but potentially elsewhere if not following such by default

Reply:
Hmm, does the tool unconditionally warn about these variables? If we cannot silence the warnings, I am fine to merge this, but in this case I request all the scripts to be checked with this tool on CI level, so that we do not violate these recommendations anywhere else in the future.

Could you update your PR and include such a check in `.travis.yml`?

Reply:
ShellCheck raises the error messages and warnings by default if that's what you mean.

In my case it is installed as a plugin in my editor (Atom).

I believe TravisCI comes with it by default but will need to read up on how to add.

I will squash the commits into one clean one and push. If I can figure the Travis thing out,  I will add it.

Otherwise will add an issue and look into dealing with it separately. 

PS. ShellCheck messages can be silenced by adding a line telling it to ignore a condition above where that condition appears.

This is needed as there can be reasons why something is done but you will obviously see the addition when reviewing.